### DISCUSSION TITLE
Multi linguage

### DISCUSSION THREAD

**Discussion by franckbertuzzi** [retypeapp/retype#494] [General]
Hello to the community

When the multi linguage will be available ?

Regards
Franck

**Comment by geoffreymcgill** [comment]
Multi-language support will be included in the upcoming release of Retype v3. 

Hope this helps.

**Reply by geoffreymcgill** [reply]
The new `locale` project config will be available in Retype v3.0, see the following comment for additional details:

See https://github.com/retypeapp/retype/issues/404#issuecomment-1469152692
